### PR TITLE
Add upload / download variable toggle

### PR DESCRIPTION
**Pull request by Undeadhunter**
Add option to enable/disable upload/download

### CODE REVIEW

**Comment by jaymoulin** [comment]
Thanks for your contribution. If you suggested it because you wanted to upload/download only, please make sure you acknowledged existence of both side projects :

# Uploader
https://github.com/jaymoulin/docker-google-music-uploader
https://hub.docker.com/r/jaymoulin/google-music-uploader/

# Downloader
https://github.com/jaymoulin/docker-google-music-downloader
https://hub.docker.com/r/jaymoulin/google-music-downloader/

--

As I understand it could be useful to transform an existing container to a downloader/uploader only without generating another key/device, I may accept this PR

File: daemon.sh [hunk 1]
```diff
@@ -19,10 +19,14 @@ else
...
-    PIDD=$!
+    if [ "$UPLOAD" = "true" ]; then
+        google-music-upload -d /media/library -a /root/oauth/oauth.key $PARAMS &
+        PIDU=$!
+    fi
+    if [ "$DOWNLOAD" = "true" ]; then
+        google-music-download -d /media/library/download -a /root/oauth/oauth.key &
+        PIDD=$!
+    fi
     wait $PIDD
```
Comment (jaymoulin) [inline daemon.sh]: Please be aware that this variable doesn't exist in all cases

File: daemon.sh [hunk 2]
```diff
@@ -19,10 +19,14 @@ else
...
     wait $PIDD
     wait $PIDU
```
Comment (jaymoulin) [inline daemon.sh]: Please be aware that this variable doesn't exist in all cases

**Review by jaymoulin — CHANGES_REQUESTED** [review]
PIDD and PIDU are now declared in a condition. Make sure those are correctly declared in other cases

File: daemon.sh (hunk 2 above)
Comment (Undeadhunter) [reply daemon.sh]: I assume adding the wait into the IF would work. 

Unless you prefer to make another IF true and put them in there?

**Comment by Undeadhunter** [comment]
Did not know those existed! Thanks!

File: daemon.sh (hunk 2 above)
Comment (jaymoulin) [reply daemon.sh]: Adding the wait in the if should work but take care of keeping the double wait (it's a feature, not a bug ;)) and name the variable `$PIDU` for uploader et `$PIDD` for download

**Comment by Undeadhunter** [comment]
All fixed

Up to you if you want to consolidate the different dockers with the ENV options.

**Comment by jaymoulin** [comment]
Do you mind update `README.md` to document this feature and squash all commits (with mine) to have a meaningful contribution?

**Comment by Undeadhunter** [comment]
That should be it, never done any squashing, so hope it's good now.

**Review by jaymoulin — APPROVED** [review]

**Comment by jaymoulin** [comment]
Thanks for your contribution ;)

**Comment by Undeadhunter** [comment]
I noticed the FALSE isn't taken correctly, will check it out later this week.

**Pull request merged**